src/lib/directives/navigation-mode.directive.ts

```diff
@@ -0,0 +1,36 @@
+import {Directive, Input, OnChanges, SimpleChanges} from '@angular/core';
+
+import {NavigationMode} from '../navigation/navigation-mode.interface';
+import {ConfigurableNavigationMode} from '../navigation/configurable-navigation-mode';
+import {WizardComponent} from '../components/wizard.component';
+
+
+@Directive({
+  selector: '[awNavigationMode]',
+})
+export class NavigationModeDirective implements OnChanges {
```

Comment:
I like it!

Still I would appreciate some comment in this class :)

Reply:
Comments added in 396face